partcad/tests/unit/test_genai.py

```
@@ -0,0 +1,49 @@
+from unittest.mock import MagicMock, patch
+
+import partcad as pc
+from partcad.part_factory_ai_cadquery import PartFactoryAiCadquery
+from partcad.part_factory_ai_openscad import PartFactoryAiScad
+
+
+def check_python_syntax(python_code):
```

Ruff (0.8.2)

Missing return type annotation for public function `check_python_syntax`

Add return type annotation: `Optional[bool]`

(ANN201)